PracticeCode/AntProjects/dividerFunction1_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from dividerFunction1 import divider
+
+# Test for dividerFunction1 pycode.
+# Test 1; func will throw an error when dividing by zero
+# Test 2; func will return a float when dividing by an Int by a float
+
+
+#divider(3,1)
+
+class TestDividerFunc(unittest.TestCase):
+    def test_zero_division_error(self):
+        # Make sure value errors is raised when int is divided by zero.
+        self.assertRaises(ZeroDivisionError, divider, 3, 0)
+
+    def test_float_result(self):
+        # capture result of divider func
+        result = divider(3, 1.0)
+        # expected result of func
+        expected_result = 3.0
+        self.assertEqual(result, expected_result)
+
+
+
+
+
+
+
+
+if __name__=="__main__":
```

Comment:
This doesn't look right. Is this the proper way to run unit tests?

Reply:
Well you can call it from the terminal with ```python3 -m unittest filename``` but in order to run it by executing the file I believe I had to add those lines.